addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -41,8 +41,8 @@ options.registry.TableOfContent = options.Class.extend({
         _.each($headings, el => {
             const $el = $(el);
             const id = 'table_of_content_heading_' + _.now() + '_' + _.uniqueId();
-            $('<a>').attr('href', "#" + id)
-                    .addClass('table_of_content_link list-group-item list-group-item-action py-2 border-0 rounded-0')
+            $('<a>').attr('href', `#${id}`)
+                    .addClass('table_of_content_link list-group-item list-group-item-action py-2 border-0 rounded-0 o_not_editable')
```

Comment:
With the `closest` change I suggested above, maybe you can then use that class on the navigation menu itself? Not sure (to see if it could be edited or not?)

Reply:
No need to add it at all! with the `closest` change it will work since the parent already [has the class](https://github.com/odoo/odoo/blob/0384903a2b06a4dbd6ab022253fbc694231b0647/addons/website/views/snippets/s_table_of_content.xml#L9) :wink: